src/dotnet/UI.Blazor.App/Components/ChatView/Items/ChatEntryMessageView.razor

```diff
@@ -59,174 +58,341 @@
     if (ReferenceEquals(entry, ChatEntry.Loading)) {
         <string-skeleton firstWidth="2" secondWidth="4" height="6" system="true"/>
     } else {
-        <div class="@cls"
-             data-hover-menu="@(m.IsSelected ? null : MenuRef.New<MessageHoverMenu>(entry.Id, bool.TrueString))"
-             data-menu="@(MenuRef.New<MessageMenu>(entry.Id, bool.TrueString, "", ""))"
-             data-menu-placement="@(FloatingPosition.BottomStart.ToPositionString())">
-            <div class="chat-message group">
-                <div class="c-content">
-                    <div class="message-content">
-                        <p class="chat-message-markup">
-                            <CascadingValue Value="@entry" IsFixed="true">
-                                <MarkupView Markup="@markup"/>
-                            </CascadingValue>
-                        </p>
+        @if (!m.EnableIncompleteUI) {
```

Comment:
please leave the comment that two blocks are almost identical and one of them must be removed after it's tests and made public